.eslintrc.js

```diff
@@ -8,6 +8,16 @@ module.exports = {
     'plugin:import/recommended',
     'plugin:import/typescript',
   ],
+  rules: {
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      {
+        argsIgnorePattern: '^_',
+        varsIgnorePattern: '^_',
+        caughtErrorsIgnorePattern: '^_',
```

Comment:
is this temporary for the skeleton function you have below? If so can you just use eslint-disable comment?

Reply:
This will be removed once everything's done. For some reason I thought that we already had this rule baked into our config somewhere, but apparently not.